plugins/com.google.cloud.tools.eclipse.appengine.libraries.test/src/com/google/cloud/tools/eclipse/appengine/libraries/model/CloudLibrariesTest.java

```diff
@@ -38,13 +38,26 @@ public void testGetLibraries() {
   public void testGetClientApis() {
     List<Library> libraries = CloudLibraries.getLibraries("clientapis");
     Assert.assertTrue(libraries.size() > 10);
+    
+    Library datalossPrevention = null;
+    Library cloudStorage = null;
+    
     for (Library library : libraries) {
       String tooltip = library.getToolTip();
+      if ("googlecloudstorage".equals(library.getId())) {
+        cloudStorage = library;
+      } else if ("datalossprevention".equals(library.getId())) {
+        datalossPrevention = library;
+      }
+      
       Assert.assertNotNull(library.getName() + " has no tooltip", tooltip);
       Assert.assertFalse(tooltip.isEmpty());
       Assert.assertFalse(library.getName() + " has no files", library.getLibraryFiles().isEmpty());
       Assert.assertEquals("clientapis", library.getGroup());
     }
+    
+    Assert.assertEquals("grpc", datalossPrevention.getTransport());
```

Comment:
These seem like they should be broken out as separate tests.

Reply:
of course, done